pkgs/development/python-modules/diffusers/default.nix

```diff
@@ -152,5 +153,8 @@ buildPythonPackage rec {
     changelog = "https://github.com/huggingface/diffusers/releases/tag/${src.rev}";
     license = licenses.asl20;
     maintainers = with maintainers; [ natsukium ];
+    # No python 3.12 support. This should be fixed in the next release.
+    # See https://github.com/huggingface/diffusers/issues/7809
+    broken = pythonAtLeast "3.12";
```

Comment:
I'd like to update `disabled` instead of `meta.broken`. Thanks.
`disabled = pythonOlder "3.8" || pythonAtLeast "3.12";`